### PR TITLE
implement HybridPlanNode/Operator workflow

### PR DESCRIPTION
**Pull request by jikunshang**

### CODE REVIEW

**Comment by jikunshang** [comment]
@Intel-bigdata/bdf-cider-engine

**Review by winningsix — COMMENTED** [review]
+1, just one quick question.

File: velox/exec/tests/HybridExecOperatorTest.cpp [hunk 1]
```diff
@@ -32,38 +31,30 @@ using facebook::velox::test::BatchMaker;
...
-                std::dynamic_pointer_cast<const HybridPlanNode>(node)) {
-          return std::make_unique<HybridExecOperator>(id, ctx, hybridOp);
-        }
-        return nullptr;
-      });
+TEST_F(HybridExecOperatorTest, hybrid) {
+  Operator::registerOperator(HybridExecOperator::planNodeTranslator);
   std::vector<RowVectorPtr> vectors;
-  for (int32_t i = 0; i < 10; ++i) {
+  for (int32_t i = 0; i < 1; ++i) {
```
[the remark this answers is not in the document]
Comment (jikunshang) [reply velox/exec/tests/HybridExecOperatorTest.cpp]: easier to debug, forget to change back.

**Comment by jikunshang** [comment]
@winningsix Please help merge this one, thanks.

**Comment by jikunshang** [comment]
based on this [PR](https://github.com/Intel-bigdata/omniscidb/pull/35)

**Pull request merged**